zerver/views/development/integrations.py

```diff
@@ -85,7 +86,7 @@ def get_fixtures(request: HttpResponse,
 def check_send_webhook_fixture_message(request: HttpRequest,
                                        url: str=REQ(),
                                        body: str=REQ(),
-                                       is_json: bool=REQ(),
+                                       is_json: bool=REQ(validator=check_bool),
                                        custom_headers: str=REQ()) -> HttpResponse:
```

Comment:
Noteworthy runtime behavior change: `?is_json=false` would previously have been interpreted as truthy.

Reply:
Noted; this tool is only used in the development environment, so should be OK.